doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -129,12 +129,16 @@ To enable one of the reactions to the last fabric removal, set the corresponding
 * :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` - Remove all saved network credentials and start Bluetooth LE advertising.
   The device will remove all saved network credentials, keep application-specific non-volatile data, and start advertising Bluetooth LE Matter service.
   After that, it will be ready for commissioning to Matter over Bluetooth LE.
+  This option is not available in the :ref:`matter_template_sample` sample.
 * :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_REBOOT` - Remove all saved network credentials and reboot the device.
   This option is selected by default.
 
   When the :kconfig:option:`CONFIG_CHIP_FACTORY_RESET_ERASE_NVS` Kconfig option is also set to ``y``, the device will also remove all non-volatile data stored on the device, including application-specific entries.
   This means the device is restored to the factory settings.
 
+To create a delay between  the chosen reaction and the last fabric being removed, set the :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_SCHEDULE_TIME` Kconfig option to a specific time in milliseconds.
```

Comment:
Btw. I noticed the links to these `CONFIG_CHIP_LAST_FABRIC_REMOVED_...` configs does not work. Could you take a look on that? For some reason links to all other configs on this page work properly.

Reply:
Ok, I know why it works like this, and the problem is bigger than I thought and not related strictly to this PR. I will create a separate task to track it.

Reply:
Right, it seems this doesn't work when we have Kconfigs defined in the Kconfig.features file...